tasks/build.py

```diff
@@ -251,8 +251,9 @@ def download_hugo(ctx):
     '''
     LOGGER.info(f'Downloading hugo version {hugo_version}')
     try:
-        dl_url = (f'https://github.com/gohugoio/hugo/releases/download/'
-                  f'v{hugo_version}/hugo_{hugo_version}_Linux-64bit.tar.gz')
+        dl_url = (f'https://github.com/gohugoio/hugo/releases/download/v'
+                  + hugo_version.split('_')[-1] +
```

Comment:
since you're already using fstrings here, we could leave this in the fstring:
```
        dl_url = (f'https://github.com/gohugoio/hugo/releases/download/'
                  f'v{hugo_version.split('_')[-1]}/hugo_{hugo_version}_Linux-64bit.tar.gz')
```